drivers/node/ssh/ssh.go

```diff
@@ -160,18 +160,17 @@ func (s *SSH) GetDeviceMapperCount(n node.Node, timerange time.Duration) (int, e
 func (s *SSH) updateDriver() error {
 	var err error
 
-	// The namespace of the portworx-service in the current context
-	execPodNamespace, err := getExecPodNamespace()
-	if err != nil {
-		return err
-	}
-	s.execPodNamespace = execPodNamespace
-	log.InfoD("UpdateDriver: ssh driver's namespace (the 'portworx-service' namespace) is updated to [%s]", execPodNamespace)
-
 	nodes := node.GetWorkerNodes()
 	if s.IsUsingSSH() {
 		err = s.initSSH()
 	} else {
+		// The namespace of the portworx-service in the current context
+		if execPodNamespace, err := getExecPodNamespace(); err != nil {
+			return err
+		} else {
+			s.execPodNamespace = execPodNamespace
+			log.InfoD("UpdateDriver: ssh driver's namespace (the 'portworx-service' namespace) is updated to [%s]", execPodNamespace)
+		}
```

Comment:
Why did we do this? We would need `s.execPodNamespace` to be set in case `s.IsUsingSSH()` is also `true`. This may affect the Px tests

Reply:
Ok, then we need to fix this.
But why by default ssh driver is checking the portworx-service.
What if the soure cluster doesn't have portworx like we are testing with pso

Reply:
Changed as discussed